unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs

```diff
@@ -24,6 +25,8 @@ public void Initialize()
             OnProfileUpdated(ownUserProfile);
 
         view.OnCloseButtonPressed += OnCloseButtonPressed;
+
+        DataStore.i.exploreV2.controller.Set(this);
```

Comment:
Why do we need to access the controller like this? `ExploreV2MenuComponentController` should know about the controller without the need of using the data store. Maybe we can just inject the controller?.

Reply:
I'm not accessing to the controller here but setting it into the DataStore in order to let other entities (in this case the Taskbar controller) access to it.

If we want not to use the DataStore to store things like controllers, how I could make the communication between ExploreV2 feature and other external entities that need the controller (like the Taskbar)?

Reply:
Ok, I have been able to communicate both features creating only data in the `DataStore`. Please take a look to: https://github.com/decentraland/unity-renderer/pull/1193/commits/41cd4726074ae8174144860be103c2887ad9a8c9.